twelvelabs/resources/embed.py

```diff
@@ -13,8 +13,12 @@
 
 
 class EmbedTask(APIResource):
-    def retrieve(self, id: str, **kwargs) -> models.EmbeddingsTask:
-        res = self._get(f"embed/tasks/{id}", **kwargs)
+    def retrieve(self, id: str, embedding_option: Optional[List[str]] = None, **kwargs) -> models.EmbeddingsTask:
+        params = {}
+        if embedding_option:
+            params["embedding_option"] = embedding_option
+        
+        res = self._get(f"embed/tasks/{id}", params=params, **kwargs)
```

Comment:
```suggestion
        params = {
            "embedding_option": embedding_option
        }
        
        res = self._get(f"embed/tasks/{id}", params=remove_none_values(params), **kwargs)
```
as we can have more params for this api, it would be great to follow current params handling way.